winml/test/scenario/cppwinrt/CustomOps.cpp

```diff
@@ -593,7 +593,7 @@ static void CustomKernelWithCustomSchema() {
     floatTensorEdgeDesc.edgeType = MLOperatorEdgeType::Tensor;
     floatTensorEdgeDesc.tensorDataType = MLOperatorTensorDataType::Float;
 
```

Comment:
@fdwr @jeffbloo please take a look at these changes too

Reply:
🤔 Looks like a custom op WinML test. @smk2007? Seems okay 🤷‍♂️ (T to T1 makes sense, given the definition above), but I'm content if the test passes.